interface/main/calendar/find_group_popup.php

```diff
@@ -0,0 +1,316 @@
+<?php
+/**
+ * interface/main/calendar/find_group_popup.php
+ *
+ * Copyright (C) 2016 Shachar Zilbershlag <shaharzi@matrix.co.il>
+ * Copyright (C) 2016 Amiel Elboim <amielel@matrix.co.il>
```

Comment:
I also should of mentioned to also keep Rod's copyright here since he had copyright on the original file.